options

```diff
@@ -0,0 +1,2 @@
+pattern_extractor="curl -sI https://assertions.ubuntu.com/_status/ping | grep Revision | perl -pe 's|X-Vcs-Revision: (\d+)|\1|'"
```

Comment:
Same about perl. Also, with sed you can use `sed -n "..../p"` so only the correct line is printed.